lib/ecto/schema.ex

```diff
@@ -133,8 +133,9 @@ defmodule Ecto.Schema do
   `{:array, inner_type}`  | `list`                  | `[value, value, value, ...]`
   `:decimal`              | [`Decimal`](https://github.com/ericmj/decimal) |
   `:map`                  | `map` |
+  `{:map, inner_type}`    | `map` |
```

Comment:
to make the API smaller, I'd consider eventually deprecating `:map` with perhaps `{:map, :any}` as a replacement.